.travis.yml

```diff
@@ -16,14 +16,8 @@ php:
 matrix:
     fast_finish: true
     include:
-        - php: '7.1'
-          env:
-              - COMPOSER_FLAGS='--prefer-lowest'
```

Comment:
I would keep one case of prefer-lowest. For now it might be a bit overkill but it's always welcome when 4.4 and 5.0 will be out